src/renderer/components/ft-playlist-add-video-prompt/ft-playlist-add-video-prompt.js

```diff
@@ -115,6 +118,12 @@ export default defineComponent({
       return this.allPlaylists.filter((playlist) => {
         if (typeof (playlist.playlistName) !== 'string') { return false }
 
+        if (this.doSearchPlaylistsWithMatchingVideos) {
+          if (playlist.videos.some((v) => v.title.toLowerCase().includes(this.processedQuery))) {
```

Comment:
![image](https://github.com/FreeTubeApp/FreeTube/assets/78101139/b5c5a320-8a4b-4457-acd5-7c5f48227ffc)
Is this intentional? Or should it be
`return playlist.videos.some((v) => v.title.toLowerCase().includes(this.processedQuery))`

Reply:
When toggle enabled, this code is an **additional** check for playlist matching the query
If return is used here, playlist without video matching query **but** playlist name matching query won't be returned

The toggle is not "search for playlist with video matching query only ignore playlist name"

Reply:
Okay sounds good, I'll approve it now